src/container/srv_target.c

```diff
@@ -855,8 +858,6 @@ cont_child_stop(struct ds_cont_child *cont_child)
 		/* cont_stop_agg() may yield */
 		cont_stop_agg(cont_child);
 		ds_cont_child_put(cont_child);
-	} else {
-		D_ASSERT(!cont_child_started(cont_child));
```

Comment:
but this is not the assertion we got right?

Reply:
oh, not this assertion, but this assertion become useless with the patch, so I removed it.